### PR TITLE
RFC: remove unsupported browser feature

### PR DESCRIPTION
**Pull request by GeoffDusome**
## What does this do/fix?

The Unsupported Browser feature in SquareOne is no longer necessary. It was built back when some browser’s couldn’t process modern JS scripts, so much so that they would crash when rendering our sites. Specifically, this feature was designed to detect and redirect version of IE9 and lower and iOS 11 or 12 (IIRC) and lower. These browsers are no longer in the wild, so we can safely remove this feature.

These feature does not address modern browsers that may not support the latest CSS features like the lack of flex gap support in iOS 14.0 . While not ideal, the site will still load and be usable in cases like this, so no fallback experience is required.

The related code has been removed from the core plugin, core theme and build system. It also removed related images to the feature. 

## QA

Links to relevant issues
- [SQONE-705](https://moderntribe.atlassian.net/browse/SQONE-705)

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because it's removing features, not adding it.
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: wp-content/themes/core/assets/css/src/theme/legacy.pcss [hunk 1]
```diff
@@ -13,5 +13,3 @@
 @import "../_mixins.pcss";
```
[the remark this answers is not in the document]
Comment (GeoffDusome) [reply wp-content/themes/core/assets/css/src/theme/legacy.pcss]: I've removed all legacy files related to styles (it was only one file). I've also removed references to the legacy file in the enqueue system. A BE dev or someone familiar with the build system might have to help me rework the changes to `Theme_Build_Parser.php` though (if necessary).

File: gulp-tasks/watch.js [hunk 2]
```diff
@@ -42,7 +42,6 @@ module.exports = {
 
 		gulp.watch( [
 			`${ pkg.square1.paths.core_theme_pcss }legacy.pcss`,
-			`${ pkg.square1.paths.core_theme_routes }unsupported_browser/css/unsupported-browser.pcss`,
 		], gulp.parallel( 'postcss:themeLegacy' ) );
```
[the remark this answers is not in the document]
Comment (GeoffDusome) [reply gulp-tasks/watch.js]: I've went through and removed all references to the legacy content in the build files. These should not appear anymore.

**Review by dpellenwood — APPROVED** [review]
Thanks, Geoff! I updated the asset handlers to remove the legacy filters and also dumped a few more legacy gulp tasks. Only remaining task is to add a changelog entry. I'd like the other staff engineers to chime in before we merge, but consider this code complete. :)

**Comment by GeoffDusome** [comment]
Weird, it looks like you had to re-remove all the things I had removed last night, oh well, looks like it all went through. Thanks for the help on the `Theme_Build_Parser.php` file, wasn't quite sure how that all worked.

**Review by ChrisMKindred — APPROVED** [review]
Looks good to me. This should help speed up development, thanks!

**Pull request merged**